sql/src/main/java/org/opensearch/sql/sql/domain/SQLQueryRequest.java

```diff
@@ -121,8 +121,11 @@ private boolean isFetchSizeZeroIfPresent() {
   }
 
   private boolean isSupportedFormat() {
-    return Strings.isNullOrEmpty(format) || "jdbc".equalsIgnoreCase(format)
-        || "csv".equalsIgnoreCase(format) || "raw".equalsIgnoreCase(format);
+    return Strings.isNullOrEmpty(format)
+            || "jdbc".equalsIgnoreCase(format)
```

Comment:
Maybe: make a enum or static array with all supported format types?

Reply:
This will be addressed in https://github.com/Bit-Quill/opensearch-project-sql/pull/217 as it is not part of legacy fallback.